backend/src/api/routes.ts

```diff
@@ -13,5 +15,9 @@ export class Routes {
       .route("/mural/:id")
       .get(this.muralController.show.bind(this.muralController))
       .put(this.muralController.update.bind(this.muralController));
+
+    app
+      .route("/route")
```

Comment:
Should this be `"/tour"`?

Reply:
Omg yeah mybad, will fix it ASAP. Thanks for pointing it out!